### PR TITLE
Bumping odo version before release

### PR DESCRIPTION
**Pull request by mohammedzee1000**
Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
 
 ### Documentation
 
-## 2.2.x
+## 2.2.4
 
 ### Feature/Enhancements
 
+- Support for `preStart` lifecycle events ([#4901](https://github.com/openshift/odo/issues/4901))
+- Do not cancel the whole execution if one of the test suite fails ([#4943](https://github.com/openshift/odo/issues/4943))
```
Comment (dharmit) [inline Changelog.md:18]:
```suggestion
- Support for `preStart` lifecycle events ([#4901](https://github.com/openshift/odo/issues/4901))
- Do not cancel the whole execution if one of the test suite fails ([#4943](https://github.com/openshift/odo/issues/4943))
- Allow linking using ServiceBinding without ServiceBindingOperator ([#4543](https://github.com/openshift/odo/issues/4543)).
```

File: Changelog.md [hunk 2]
```diff
@@ -1,28 +1,32 @@
 #  Odo Changelog
 
-## Next
+## 2.3.0
```
Comment (dharmit) [inline Changelog.md:3]: We're not sure when 2.3.0 will be released. Can we not put this here? `Next` sounds apt even though it was me who was confused. 😀

**Review by dharmit — CHANGES_REQUESTED** [review]

File: Changelog.md [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
+
 - Fix component create and delete help examples([#4904](https://github.com/openshift/odo/pull/4904))
+- `odo push` now displays "Waiting for component to start" only once ([4919](https://github.com/openshift/odo/pull/4919))
+- Modifies the error message to list the available starters ([#4935](https://github.com/openshift/odo/pull/4935))
 
 ### Tests
 
+- Test storage + link ([#4954](https://github.com/openshift/odo/pull/4954))
+
 ### Documentation
```
Comment (valaparthvi) [inline Changelog.md:31]:
```suggestion
### Documentation

- Add Getting Started > Basics documentation ([#4914](https://github.com/openshift/odo/pull/4914))
- Add documentation for global configuration ([#4921](https://github.com/openshift/odo/pull/4921))
- Add odo installation docs ([#4923](https://github.com/openshift/odo/pull/4923))
```

File: Changelog.md [hunk 4]
```diff
@@ -1,28 +1,32 @@
...
 ### Feature/Enhancements
 
+- Support for `preStart` lifecycle events ([#4901](https://github.com/openshift/odo/issues/4901))
+- Do not cancel the whole execution if one of the test suite fails ([#4943](https://github.com/openshift/odo/issues/4943))
+
 ### Bug Fixes
+
 - Fix component create and delete help examples([#4904](https://github.com/openshift/odo/pull/4904))
+- `odo push` now displays "Waiting for component to start" only once ([4919](https://github.com/openshift/odo/pull/4919))
+- Modifies the error message to list the available starters ([#4935](https://github.com/openshift/odo/pull/4935))
```
Comment (valaparthvi) [inline Changelog.md:26]: Add to the list: 
```md
- Add language and projectType metadata in devfile for s2i components([#4911](https://github.com/openshift/odo/pull/4911))
```

File: Changelog.md (hunk 2 above)
Comment (mohammedzee1000) [reply Changelog.md:3]: That was the reason i put next there infact. Next for next version, and that will be updated by version bump pr

File: Changelog.md (hunk 1 above)
Comment (valaparthvi) [reply Changelog.md:18]: Also, add the following:
```markdown
- Add Sonar Cloud config file: sonar-project.properties([#4900](https://github.com/openshift/odo/pull/4900))
```

**Review by valaparthvi — CHANGES_REQUESTED** [review]
Please added the features, bugs, and docs that I worked on.

File: Changelog.md (hunk 4 above)
Comment (mohammedzee1000) [reply Changelog.md]: Added

File: Changelog.md (hunk 3 above)
Comment (mohammedzee1000) [reply Changelog.md:32]: Updated

File: Changelog.md (hunk 1 above)
Comment (valaparthvi) [reply Changelog.md:18]: @mohammedzee1000 Missed out on this one.

**Comment by valaparthvi** [comment]
https://github.com/openshift/odo/pull/4966#discussion_r682362608
@mohammedzee1000 You probably missed this one.

File: Changelog.md (hunk 1 above)
Comment (mohammedzee1000) [reply Changelog.md:18]: updated, thanks

**Comment by mohammedzee1000** [comment]
Windows timeout:


/test psi-unit-test-windows

**Review by dharmit — COMMENTED** [review]
/approve

@valaparthvi can you add LGTM if it looks good to you?

**Review by valaparthvi — APPROVED** [review]

**Pull request merged**